priv/repo/migrations/20160802030642_create_note.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Notox.Repo.Migrations.CreateNote do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes) do
+      add :title, :string
+      add :description, :string
+
+      timestamps
```

Comment:
Hmm. Nope. I've definitely used it before, and I can see an example of it being used here: https://github.com/elixir-ecto/ecto/blob/599d4a666ccbb7af48988349611938b6ff1606ab/integration_test/support/migration.exs#L25

When you say it "doesn't work", what does that mean?